Packs/Base/ReleaseNotes/1_0_3.md

```diff
@@ -0,0 +1,4 @@
+
+### Scripts
+- __CommonServerPython__  
+Updated `handle_proxy` function to unset relevant environment variables when *unsecure* or *insecure* params are set.
```

Comment:
Should add also to the `CHANGELOG.md` file

Reply:
@guyfreund Is correct. At least for now, the `CHANGELOG.md` is required.

Release notes look great.